deepes/src/paddle/async_es_agent.cc

```diff
@@ -0,0 +1,258 @@
+//   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "async_es_agent.h"
+namespace DeepES {
+
+AsyncESAgent::AsyncESAgent(
+    std::shared_ptr<PaddlePredictor> predictor,
+    std::string config_path): ESAgent(predictor, config_path) {
+  _config_path = config_path;
+}
+AsyncESAgent::~AsyncESAgent() {
+  for(const auto kv: _param_delta) {
+    float* delta = kv.second;
+    delete[] delta;
+  }
+}
+
+bool AsyncESAgent::_save() {
+  bool success = true;
+  if (_is_sampling_agent) {
+    LOG(ERROR) << "[DeepES] Original AsyncESAgent cannot call `save`.Please use cloned AsyncESAgent.";
```

Comment:
It should be ""[DeepES] Cloned AsyncESAgent cannot call `save`.Please use original AsyncESAgent.""

Reply:
done